invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js

```diff
@@ -28,6 +28,10 @@ export const Results = ({ currentResultsState = {} }) => {
   const { sortOptions, layoutOptions, paginationOptions, buildUID } =
     useContext(SearchConfigurationContext);
   const multipleLayouts = layoutOptions.listView && layoutOptions.gridView;
+
+  const handleMathjaxCallback = useCallback(() => {
+    window.MathJax?.typeset();
```

Comment:
What I don't find very elegant here, and in general, is the hardcoding of the MathJax calls.
It would be more elegant to have a series of configurable plugins, that are called onRender. It happens that in `InvenioRDM` (and not in Invenio modules), the only configured plugin is MathJax.